serverless.yml

```diff
@@ -0,0 +1,60 @@
+plugins:
+  - serverless-offline
+  - serverless-python-requirements
+  - serverless-plugin-tracing
+  - serverless-prune-plugin
+  - serverless-latest-layer-version
+
+service: eth-event-listener
+provider:
+  name: aws
+  runtime: python3.7
+  description: Ethereum event listener
+  memorySize: 512
+  timeout: 90
+  region: ${file(./config.${self:provider.stage}.json):REGION}
+  stage: ${opt:stage,'dev'}
+  deploymentBucket:
+    name: snet-serverless-artifacts
+    serverSideEncryption: AES256
+    tags:
+      key1: registry
+    deploymentPrefix: serverless
+    tracing: true
+
```

Comment:
- I don't see test case
- no CI configured (code coverage, no test, code report)

Reply:
Circleci and test cases are checked in. We will configure the circleci project once @vinthedark is back.